relentless/optimize/method.py

```diff
@@ -391,3 +528,88 @@ def line_search(self, value):
         if value is not None and not isinstance(value, LineSearch):
             raise TypeError('If defined, the line search parameter must be a LineSearch object.')
         self._line_search = value
+
+class FixedStepDescent(SteepestDescent):
+    r"""Fixed-step steepest descent algorithm.
+
+    For an :class:`~relentless.optimize.objective.ObjectiveFunction` :math:`f\left(\mathbf{x}\right)`,
+    the fixed-step steepest descent algorithm seeks to approach a minimum of the
+    function.
+
+    :math:`\mathbf{X}=\left[X_1,\ldots,X_n\right]` is defined as the scaling
+    parameters for the gradient such that :math:`y_i=\frac{x_i}{X_i}`. By
+    default, the variables are left unscaled (:math:`X_i=1`).
+
+    With scaling, the gradient of the function becomes:
+
+    .. math::
+
+        \nabla f\left(\mathbf{y}\right) = \left[X_1 \frac{\partial f}{\partial x_1},
+                                                      \cdots,
+                                                X_n \frac{\partial f}{\partial x_n}\right]
+
+    :math:`\alpha` is defined as the step size hyperparameter. The step size must
+    be specified numerically, and a :class:`LineSearch` can be performed to find
+    an optimal step size value if desired.
+
+    Then, with an initial numerical guess for all the design variables :math:`\mathbf{x}`,
+    the following iterative calculation is performed:
+
+    .. math::
+
+        \mathbf{y}_{n+1} = \mathbf{y}_{n}-\alpha\frac{\nabla f\left(\mathbf{y}\right)}
+                                                     {\lVert\nabla f\left(\mathbf{y}\right)\rVert}
```

Comment:
Lumping the norm with alpha makes the intent clearer:
```suggestion
        \mathbf{y}_{n+1} = \mathbf{y}_{n}-\frac{\alpha}{\lVert\nabla f\left(\mathbf{y}\right)\rVert} \nabla f\left(\mathbf{y}\right)
```